lib/modules/datasource/nuget/v3.ts

```diff
@@ -242,3 +267,31 @@ export async function getReleases(
 
   return dep;
 }
+
+async function getSourceUrlFromNupkg(
+  http: Http,
+  packageName: string,
+  packageVersion: string | null,
+  nupkgUrl: string,
+): Promise<string | undefined> {
+  const cacheDir = await ensureCacheDir(`nuget`);
```

Comment:
add an experimental environment variable to disable this. as told before I need this disabled.